frontend/src/components/TestedPerDayPositivePercentLineChart.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  import c3, { ChartConfiguration, ChartAPI, PrimitiveArray } from "c3";
+
+  import {
+    formatNumber,
+    getChartConfig,
+    getTestedPerDayPercentChartRows,
+  } from "../utils";
+  import { infectedIncreaseLog } from "../store";
+  import ChartLayout from "./ChartLayout.svelte";
+
+  export let title: string;
+  export let id: string;
+
+  let timespan = 14;
+  let rows: PrimitiveArray[] = [];
+  let chart: ChartAPI | undefined;
+
+  function tooltipValue(
+    value: number,
+    ratio: number | undefined,
+    id: string,
+    index: number
+  ): string {
+      (rows[index + 1][1] as number) + (rows[index + 1][2] as number);
+    const percentage = Math.round(value * 10000) / 100;
+    return `${percentage}%)`;
+  }
+
+  function labelsFormatter(value: number): string {
+    if (timespan === -1) {
+      return "";
+    }
+    return formatNumber(value);
+  }
+
+  function getTPDLineChartConfig(timespan: number): ChartConfiguration {
+    const newRows = getTestedPerDayPercentChartRows($infectedIncreaseLog, timespan);
+    // Replacing rows contens so tooltip formatter works correctly
+    rows.splice(0, Infinity, ...newRows);
+    return getChartConfig({
+      bindto: `#${id}_graph`,
+      data: {
+        rows,
+        labels: { format: labelsFormatter },
+      },
+      tooltip: {
+        format: {
+          value: tooltipValue,
+        },
+      },
```

Comment:
Please add y axis tick formatting. Currently there is 0.2 instead of 20%
```
y: {
  tick: {
    format: (value) => {/*float value to percent string*/}
  }
}
```

Reply:
Added